Content rekenen met breuken en kommagetallen (#41)
florisdf/hoe-zit-het: content/lessen/wiskunde/rekenen_rationaal/breuken/index.md

---
title: "Wat is een breuk?"
date: 2020-07-05T15:00:00+02:00
weight: 2
draft: true
images: []
tags: ["Breuk", "Rationale getallen", "Bewerkingen", "Waarom kan je niet delen door 0?"]
description: "In deze les zoeken we uit wat een breuk precies is."
---
Je bent op een feestje met acht personen en het is tijd voor de taart :birthday:. Iedereen wil uiteraard een even groot
stuk van de taart, dus wordt hij in acht gelijke stukken gesneden :cake:. Iedereen krijgt dus één achtste of $\frac{1}{8}$ van de taart.

> 

Reviewer: ```suggestion

Zoals je wellicht al weet, noemen we die $\frac{1}{8}$ een **breuk**. In deze les leggen we uit wat zo'n breuk weer precies is en in welke vormen een breuk zoal kan voorkomen.

```

> ## Waar staat de teller?

Reviewer: ```suggestion
## Een breuk bestaat uit een teller en een noemer
```

Author: Wordt er in Google niet meer gezocht op 'Waar staat de teller?'. Ik ben toch al enkele keren schuldig geweest aan die zoekterm!

> 

Reviewer: ```suggestion

Een breuk bestaat uit een **teller** en een **noemer**. Tussen de teller en noemer trekken we een **breukstreep**.

```

{{< svg "img/terminologie.svg" "Een breuk bestaat uit een breukstreep met daarboven de teller. Onder de breukstreep staat de noemer." >}}

> De **teller** *telt* het aantal delen dat je neemt. De **noemer** *benoemd* hoe groot het deel is :exploding_head:. Je kan onthouden: de **t**eller staat aan de **t**op! 

Reviewer: ```suggestion
De **teller** *telt* het aantal stukken dat je neemt. De **noemer** *benoemd* in hoeveel stukken we de taart hebben verdeeld. In de breuk $\frac{3}{4}$ is de teller $3$ en de noemer $4$. Dat betekent dat we de taart in $4$ stukken hebben verdeeld en we daar $3$ stukken van nemen.

Een trucje om de teller en noemer te onthouden: de **t**eller staat aan de **t**op! 
```

| Je schrijft       | Je leest het als               |
|-------------------|--------------------------------|

> | $\frac{a}{b}$     | a over b                       |

Reviewer: ```suggestion
| $\frac{3}{4}$     | drie vierde |
| $\frac{1}{8}$     | één achtste |
| $\frac{5}{4}$     | vijf vierde  |
| $\frac{2}{3}$     | twee derde |
```

> ## Hoe werkt een breuk?

Reviewer: Deze paragraaf is misschien iets te veel "van de hak op de tak". Probeer om per paragraaf duidelijk af te lijnen waar ze over gaat. Dat zou je in één zin heel concreet moeten kunnen zeggen en misschien is "hoe en breuk werkt" iets te algemeen. Het lijkt me ook dat de eerste alinea's kunnen samengevoegd worden met de vorige paragraaf. Bv. de volgende paragrafen:

1. Een breuk bestaat uit een teller en een noemer -> Waar staan teller en noemer in een breuk en hoe kan je ze interpreteren?
2. Een breuk als deling -> Een breuk is eigenlijk een deling
3. Tekens in een breuk -> Een minteken kan je doorheen de breuk verschuiven. Twee mintekens mag je schrappen.
4. Gelijkwaardige en gelijknamige breuken -> Gelijkwaardige breuk zijn breuken die gelijk zijn aan elkaar, gelijknamige breuken zijn breuken met dezelfde noemer
5. Breuken, kommagetallen en procenten als rationale getallen -> Breuken, (rationale) kommagetallen en procenten zijn inwisselbaar. Ze zijn allemaal elementen van de rationale getallen.

> Het is handig om bij breuken **een pizza** :pizza: voor te stellen. De **noemer** van een breuk vertelt je **in hoeveel stukken** de pizza wordt verdeeld.

Reviewer: ```suggestion
Niet schrikken als je tijdens een les over breuken plots honger krijgt. Je kan een breuk namelijk eenvoudig voorstellen met bijvoorbeeld taarten of pizza's. :pizza: Zoals we hierboven al zeiden, vertelt de **noemer** van een breuk **in hoeveel stukken** je de pizza {{< mute "(of taart)" >}} verdeelt.
```

De **teller** van een breuk vertelt je dan **hoeveel stukken je neemt/eet** :yum:. 

> Zo is de pizza hieronder verdeelt in vier **gelijke** stukken. De *kleine* pizzapunt

Reviewer: ```suggestion
Zo is de pizza hieronder verdeeld in vier **gelijke** stukken. De *kleine* pizzapunt
```

stelt $\frac{1}{4}$ voor, terwijl de rest van de pizza $\frac{3}{4}$ voorstelt.

{{< svg "img/pizza.svg" "Een pizza die in vier stukken is gedeeld, waarvan eentje wordt genomen." >}}

> Een andere manier om aan breuken te denken is door middel van **een deling**. Zo betekent $\frac{3}{4}$ hetzelfde als $3 : 4$. De uitkomst van beide is $0,75$. Het is **veel makkelijker** om een **deling als een breuk te schrijven**, daarom werken wiskundigen veel liever met breuken in plaats van het deelteken.

Reviewer: ```suggestion
Een andere manier om aan breuken te denken is door middel van **een deling**. Zo betekent $\frac{3}{4}$ hetzelfde als $3 : 4$. De uitkomst van beide is $0{,}75$. In veel gevallen is het duidelijker wanneer we een **deling als een breuk te schrijven**. Neem bijvoorbeeld onderstaande bewerkingen

$$5 : 4 + 3 : 2$$
$$\frac{5}{4} + \frac{3}{2}$$

Beide berekeningen zijn eigenlijk hetzelfde, maar het geval van de breuken is veel overzichtelijker. Daarom zullen we er meestal voor kiezen om een deling als een breuk te schrijven.
```

> Aangezien een breuk eigenlijk een deling is, moeten we nog even naar onze **mintekens** kijken :eyes:. Zowel teller als noemer kunnen één (of meer) minteken(s) bevatten. Als **alleen de teller** of **alleen de noemer negatief** is, dan is de **breuk in zijn geheel negatief**. We noteren een minteken voor de breukstreep ($-\frac{a}{b}$). Als **teller én noemer negatief** zijn, dan is de **breuk in zijn geheel positief**.

Reviewer: Misschien hiervoor een aparte paragraaf maken?

Hieronder vind je een tabel die toont hoe de rechterlimiet (die nadert van rechts) wordt berekend. Voor de linkerlimiet voeren we hetzelfde proces uit, maar nemen we steeds een negatieve $x$.

| $x$     | $\frac{1}{x}$     |
|---------|-------------------|
| 1       | 1                 |
| 0.1     | 10                |
| 0.01    | 100               |
| 0.001   | 1000              |
| 0.0001  | 10000             |
| 0.00001 | 100000            |
| ...     | ...               |
| 0       | $+\infty$         |

Het is dus duidelijk dat we **geen eenduidige oplossing** hebben wanneer we **delen door 0**. Vandaar dat dit **ongedefinieerd** is.

> 

Reviewer: Een heel goede uitleg, maar ik vrees dat een leerling van het eerste middelbaar iets te weinig kaas gegeten heeft van Analyse om het te kunnen begrijpen ;-) Ik zou het achterwege laten

Author: Maar voor zij die het begrijpen, volledig te zijn en een sneakpeek te geven in de Analyse... Want waar moeten we dat anders zetten (aangezien dat dat toch de meest correcte reden is...)

{{< /expand >}}

> ## Soorten breuken

Reviewer: ```suggestion
## Gelijkwaardige en gelijknamige breuken
```

> Er bestaan verschillende **speciale breuken** die hun eigen naam hebben gekregen.

Reviewer: Bij het lezen van deze zin, dacht ik dat je breuken als "een kwart" of "de helft" bedoelde. "Eigen naam" is een beetje verwarrend denk ik

> Sommige breuken zien er anders uit, maar betekenen eigenlijk hetzelde. Zo'n breuken noemen we **gelijkwaardige breuken**. Of ze nu zeggen dat je $\frac{1}{2}$ van de pizza krijgt of $\frac{2}{4}$, uiteindelijk zal je dezelfde hoeveelheid pizza kunnen eten. We kunnen namelijk zowel de teller als de noemer van $\frac{2}{4}$ **delen door {{< class "2" "orange" >}}**. Dan krijgen we $\frac{1}{2}$.

Reviewer: ```suggestion
Sommige breuken zien er anders uit, maar zijn eigenlijk gelijk aan elkaar. Zulke breuken noemen we **gelijkwaardige breuken**. Of ze nu zeggen dat je $\frac{1}{2}$ van de pizza krijgt of $\frac{2}{4}$, uiteindelijk zal je dezelfde hoeveelheid pizza kunnen eten. We kunnen namelijk zowel de teller als de noemer van $\frac{2}{4}$ **delen door {{< class "2" "orange" >}}**. Dan krijgen we $\frac{1}{2}$.
```

$$\frac{\frac{2}{\orange{2}}}{\frac{4}{\orange{2}}} = \frac{1}{2}$$

{{< attention "Handig!" >}}
Als je een breuk hebt, mag je steeds de **teller en noemer vermenigvuldigen of delen met eenzelfde getal**.

$$\frac{25}{40} = \frac{\frac{25}{\orange{5}}}{\frac{40}{\orange{5}}} = \frac{5}{8}$$
$$\frac{1}{2} = \frac{1 \cdot \orange{10}}{2 \cdot \orange{10}} = \frac{10}{20}$$

{{< /attention >}}

Een andere soort breuken die we vaak tegenkomen zijn de **gelijknamige breuken**. Gelijknamige breuken hebben **dezelfde noemer**. Enkele voorbeelden hiervan zijn:

$$\frac{2}{\green{5}}, \frac{8}{\green{5}}, \frac{25}{\green{5}}$$

> Laten we het laatste voorbeeld nog eens van dichterbij bekijken :mag_right:. We kunnen $\frac{25}{5}$ namelijk ook schrijven als $\frac{5}{1}$ {{< mute "of simpelweg als 5." >}} Als we dit doen is de breuk **niet meer gelijknamig** met de anderen, want de **noemer is verandert**. 

Reviewer: ```suggestion
Laten we het laatste voorbeeld nog eens van dichterbij bekijken :mag_right:. We kunnen $\frac{25}{5}$ namelijk ook schrijven als $\frac{5}{1}$ {{< mute "of simpelweg als 5." >}} Als we dit doen is de breuk **niet meer gelijknamig** met de anderen, want de **noemer is veranderd**. 
```
